fabric-rendering-v1/src/main/java/net/fabricmc/fabric/mixin/client/rendering/MixinBuiltinModelItemRenderer.java

```diff
@@ -0,0 +1,26 @@
+package net.fabricmc.fabric.mixin.client.rendering;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.item.BuiltinModelItemRenderer;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.fabric.api.client.rendering.v1.BuiltinItemRenderer;
+import net.fabricmc.fabric.impl.client.rendering.BuiltinItemRendererRegistryImpl;
+
+@Mixin(BuiltinModelItemRenderer.class)
+class MixinBuiltinModelItemRenderer {
+	@Inject(method = "render", at = @At("RETURN"))
+	private void fabric_onRender(ItemStack stack, MatrixStack matrices, VertexConsumerProvider vertexConsumers, int light, int overlay, CallbackInfo info) {
```

Comment:
Hmm, should this injection be at head and return early instead? I don't see a point of having it in tail (i.e. waiting for all the vanilla if branch checks)

Reply:
Yeah, that seems good.

Reply:
Well that definitely branches onto an argument of, what if someone wanted to replace a built-in renderer for vanilla items, i.e. change the shulker box model